### DISCUSSION TITLE
Download Error

### DISCUSSION THREAD

**Discussion by rumphelstilskin** [aclap-dev/video-downloadhelper#619] [Help and Q&A]
### Your question

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://dizipal807.com/bolum/gibi-1x1",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : Not detecting m3u8/hls with non standard extension and non standard mime type\r\n    Last message repeated 6 times\r\n[in#0 @ 00000267f785b180] Error opening input: Invalid data found when processing input\r\nError opening input file https://dplayer74.site/l.php?v=ZThNY3Jnb2NDZDJGcWlvejdyN1ZyM0cyMzBtVXlNLzdiQkRaSG1lN0c4U1RPaitEb0dmVjBvYXJ5WlBEeC81RXNaNHVDZnhYcngrNm5Zbi94bzFyOWgxalB4S2RMWW93V0lSbFJGcHVlWFB4QTdpOWhlMHRnNFdOMWFWdHlaVWdOaWt5aTJDL3hTYnBhajVoZUkyUHJCUTFFVjdXOXovUmxpNlV6UW5ZNUdjcjZ2MDN6QkQyOXpqeXJaOG5lS29NdnBmVy9EQXlBM2ZMakVsSXdVRFhhZHpVREZWNGZzSW1nWFpPeGhseHdUT3N0M0MvbnphNm9lZGZ6bm1xY0tvVk9pcWlzMmRKNDdiZlY1UkdON1ovdWlXQVVoKzRVb2NMQ3VYRlRwb3E5aW5aR01ReHBadlF0YVVvampNbjh1V1pTaTJpeGRVL0huZnMrSGNROGN3N21LSEZCNEpBRThpa1hjOUpuNXlGMlJXZlg4UzNtY3A1Y2NCVzNRZHN1VnppT1Y1d0RmZll3dlIralZVNitHbjlYSEtsdXlDSGg1ZTZqbEU1SEhoNno2dVo3RmJxZm5PMklFWnRxdk9TOXRFSC9jM05NNGdWK0hTV0RPT2F1OTF2ZXQ2RGh3WVcrYUE2NHNqRy9oWWwrYW1pZWZIbWhkd3JNSDg4a3U5N3NuUHhadGhUOVpxbGN3OXM2UGlqcGdQV1Z4bkdTcURhN1VoSE04Zmx3bTN5d1F5MFpDejNxK2lBcTVtL2YrUXR1TE5hZTFQbVZMOW1nZGRnd28rTnpsS1hGZGJua0J2bDB5djdKaGlqSzdEdnRLa0JGYlFqTW9lbU8zblBibmlrbGYvS25QWjAvTmoxVUJzYTRUaExZenQ2RzhTbWsycnk5Tno5L1NMMy9ZSm9hdkdpRnV4ejY1RlZjblJzSFg4RS9kZ2dYOVNaMHhLZFFQRXMwRUhjSDBtM0J5V3hMMERjM2dpTEFNUExQOE5DT1Z4Sld4a1owQVZ0U2FRU2NWekhPMHF5NUVDdG9aRzhFRW42bElzYk9KMDl3Q2RPcTJqS1l1ZXlqc0tVQ3d2ejdKQ24yWnBwa1dEdG13N09naXlBUFppeStqY0FWNXJuVUlyeUlqVkFBWGRLNGF5LzZjNnlUcWFhZS85Y3lzUE9uNEo2NzFwdzBYekJuV2E1Rk9yZnZ2M0hrMTVBVEZ3eDNpOHFKZmlkTWVxeVllMWU4K3JSSmlUU0hyd0VveXNhLzFybFY0Z2gvKy8rQytwK051SkZGdThGeHhUMjZlU0hiRnBzdUZGRjJYeXIrUnRjem9mMkJqOG53VFZ5ZTFmUmhDQnhYMW9ORU1kMVAvRnN2SDVXYXF6K2xaRHNGNmxoUGhwRW9XRUdEVmdPcFZHMklucDE3eHIvL205MWJGSTR5dkRyZXFTUDk1OWYvajBnMmFCOXk5UWYwcHZZcDBFZVBKd3A2TFdLVCtid2h1SFE0VksxdDExcEp6U3pUQWF4K0hrMUNtbm44NmRldnVUdER4OTNRYzhqeUNPdVNXT01wSTlnUDFTanRFTDNudE5YY0lUZVN1MGRPOHVSZmRJWERLRi94MDVxUVExNldKVUFtcGcrY1VRRnVxYWFmQkdteGY4VjNQUnNMMGdZSEFTa2JLbHJ1bjVoaXY1M3YxWEpRR009.\r\nError opening input files: Invalid data found when processing input\r\n",
    "id": "coapp_failure_ff549447-eed9-4574-b0b4-3b9de0f80b29",
    "downloadable_id": "downloadable_8476308309419529"
  }
}

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://dizipal807.com/bolum/gibi-1x1",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : Not detecting m3u8/hls with non standard extension and non standard mime type\r\n    Last message repeated 6 times\r\n[in#0 @ 00000267f785b180] Error opening input: Invalid data found when processing input\r\nError opening input file https://dplayer74.site/l.php?v=ZThNY3Jnb2NDZDJGcWlvejdyN1ZyM0cyMzBtVXlNLzdiQkRaSG1lN0c4U1RPaitEb0dmVjBvYXJ5WlBEeC81RXNaNHVDZnhYcngrNm5Zbi94bzFyOWgxalB4S2RMWW93V0lSbFJGcHVlWFB4QTdpOWhlMHRnNFdOMWFWdHlaVWdOaWt5aTJDL3hTYnBhajVoZUkyUHJCUTFFVjdXOXovUmxpNlV6UW5ZNUdjcjZ2MDN6QkQyOXpqeXJaOG5lS29NdnBmVy9EQXlBM2ZMakVsSXdVRFhhZHpVREZWNGZzSW1nWFpPeGhseHdUT3N0M0MvbnphNm9lZGZ6bm1xY0tvVk9pcWlzMmRKNDdiZlY1UkdON1ovdWlXQVVoKzRVb2NMQ3VYRlRwb3E5aW5aR01ReHBadlF0YVVvampNbjh1V1pTaTJpeGRVL0huZnMrSGNROGN3N21LSEZCNEpBRThpa1hjOUpuNXlGMlJXZlg4UzNtY3A1Y2NCVzNRZHN1VnppT1Y1d0RmZll3dlIralZVNitHbjlYSEtsdXlDSGg1ZTZqbEU1SEhoNno2dVo3RmJxZm5PMklFWnRxdk9TOXRFSC9jM05NNGdWK0hTV0RPT2F1OTF2ZXQ2RGh3WVcrYUE2NHNqRy9oWWwrYW1pZWZIbWhkd3JNSDg4a3U5N3NuUHhadGhUOVpxbGN3OXM2UGlqcGdQV1Z4bkdTcURhN1VoSE04Zmx3bTN5d1F5MFpDejNxK2lBcTVtL2YrUXR1TE5hZTFQbVZMOW1nZGRnd28rTnpsS1hGZGJua0J2bDB5djdKaGlqSzdEdnRLa0JGYlFqTW9lbU8zblBibmlrbGYvS25QWjAvTmoxVUJzYTRUaExZenQ2RzhTbWsycnk5Tno5L1NMMy9ZSm9hdkdpRnV4ejY1RlZjblJzSFg4RS9kZ2dYOVNaMHhLZFFQRXMwRUhjSDBtM0J5V3hMMERjM2dpTEFNUExQOE5DT1Z4Sld4a1owQVZ0U2FRU2NWekhPMHF5NUVDdG9aRzhFRW42bElzYk9KMDl3Q2RPcTJqS1l1ZXlqc0tVQ3d2ejdKQ24yWnBwa1dEdG13N09naXlBUFppeStqY0FWNXJuVUlyeUlqVkFBWGRLNGF5LzZjNnlUcWFhZS85Y3lzUE9uNEo2NzFwdzBYekJuV2E1Rk9yZnZ2M0hrMTVBVEZ3eDNpOHFKZmlkTWVxeVllMWU4K3JSSmlUU0hyd0VveXNhLzFybFY0Z2gvKy8rQytwK051SkZGdThGeHhUMjZlU0hiRnBzdUZGRjJYeXIrUnRjem9mMkJqOG53VFZ5ZTFmUmhDQnhYMW9ORU1kMVAvRnN2SDVXYXF6K2xaRHNGNmxoUGhwRW9XRUdEVmdPcFZHMklucDE3eHIvL205MWJGSTR5dkRyZXFTUDk1OWYvajBnMmFCOXk5UWYwcHZZcDBFZVBKd3A2TFdLVCtid2h1SFE0VksxdDExcEp6U3pUQWF4K0hrMUNtbm44NmRldnVUdER4OTNRYzhqeUNPdVNXT01wSTlnUDFTanRFTDNudE5YY0lUZVN1MGRPOHVSZmRJWERLRi94MDVxUVExNldKVUFtcGcrY1VRRnVxYWFmQkdteGY4VjNQUnNMMGdZSEFTa2JLbHJ1bjVoaXY1M3YxWEpRR009.\r\nError opening input files: Invalid data found when processing input\r\n",
    "id": "coapp_failure_ff549447-eed9-4574-b0b4-3b9de0f80b29",
    "downloadable_id": "downloadable_8476308309419529"
  }
}

**Comment by Mattmjs** [comment]
I think this is a site restricted to Turkey if I remember from the old google groups forum. They were using a super stealthy streaming
method. I couldn't find how it was streamed & the segments were playing as png files. So I don't know that VDH will work for
this site.
